src/MTGCore/Controllers/CardController.cs

```diff
@@ -16,18 +18,20 @@ public class CardController : Controller
     {
         private MTGService _mtgService;
         private IMapper _mapper;
+        private readonly RepoContext _context;
 
-        public CardController(MTGService mtgservice, IMapper mapper)
+        public CardController(MTGService mtgservice, IMapper mapper, RepoContext context)
```

Comment:
Would it be worth passing in an IRepoContext interface here that RepoContext inherits to ensure it implements the required methods?

Reply:
done.